torch/types.py

```diff
@@ -70,8 +70,9 @@
 
 # Meta-type for "device-like" things.  Not to be confused with 'device' (a
 # literal device object).  This nomenclature is consistent with PythonArgParser.
+ExplicitDevice: TypeAlias = Union[_device, str, int]
 # None means use the default device (typically CPU)
-Device: TypeAlias = Union[_device, str, int, None]
+Device: TypeAlias = Union[ExplicitDevice, None]
```

Comment:
```suggestion
OptionalDevice: TypeAlias = Union[ExplicitDevice, None]
Device: TypeAlias = OptionalDevice
```

nit: I think having an `OptionalDevice` compared to `ExplicitDevice`.

Reply:
I can't find a good name..